package.json

```diff
@@ -17,7 +17,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint:fix": "eslint --fix --ext .js ."
```

Comment:
Added a script here to fix lint problems, but I could add a pre-commit hook here if we deem it necessary.

Reply:
That would be a nice to have.